package.json

```diff
@@ -36,8 +36,8 @@
     "body-parser": "1.10.0",
     "bookshelf": "0.7.9",
     "busboy": "0.2.9",
+    "chalk": "^1.0.0",
```

Comment:
Can you change this to `"chalk": "1.0.0",` so it's pegged to a specific version like the rest of the dependencies?  Thanks.